bionlp/taggers/rnn_feature/tagger_utils.py

```diff
@@ -123,6 +123,41 @@ def trim_tags(tagged_data):
                           for x, y in tagged_data[i]]
     return tagged_data
 
+def get_regular_or_capitalized_embeddings(w2i, i2w, mdl, params):
+    embedding_vector_size = params['emb1_size']
+    embeddings = []
+    for i in range(len(w2i)):
+        term_emb = None
+        if i in i2w:
+            regular_term = str(i2w[i])
+            capitalized_term = regular_term.capitalize()
+            if regular_term in mdl:
+                term_emb = mdl[regular_term]
+            elif capitalized_term in mdl:
+                term_emb = mdl[capitalized_term]
+
+        if term_emb == None:
+            term_emb = np.zeros(embedding_vector_size,)
+
+        embeddings.append(term_emb)
+    return np.array(embeddings)
+
+def create_vocab_output_files(emb_i, i2w, zero_vectors_out_file, nonzero_vectors_out_file):
```

Comment:
I used locally to create two text files:

- the first one contains a list of all terms that have been initialized with an all-zero vector
- the second one contains a list of all terms that have a corresponding word embedding vector in `mdl` and are thus initialized with that vector

These files have simply been useful as an overview for me, getting an insight into which terms are created using the word embedding matrix and which are not. I considered the function useful for future use and to be correctly placed in the `tagger_utils` module, so I left it there.